spring-pulsar/src/main/java/org/springframework/pulsar/annotation/PulsarListener.java

```diff
@@ -186,4 +186,12 @@
 	 */
 	AckMode ackMode() default AckMode.BATCH;
 
+	/**
+	 * The bean name or a 'SpEL' expression that resolves to a
+	 * {@link org.springframework.pulsar.listener.PulsarConsumerErrorHandler} which is
+	 * used as a Spring provided mechanism to handle errors from processing the message.
+	 * @return the bean name for the consumer error handler or an empty string.
+	 */
+	String pulsarConsumerErrorHandler() default "";
```

Comment:
So we will have the default error handler auto-configured, correct? And to use that one the users would just set "beanNameOfWhatTheDefaultIsCalledInOurAutoConfig" ?